tests/integration/test_get_item.py

```diff
@@ -36,7 +36,7 @@ def test_basic():
 def test_scalar_ndarray_as_index(arr):
     offsets = num.arange(5)  # [0, 1, 2, 3, 4]
     offset = offsets[3]  # 3
-    # assert arr[offset] == 1  # TODO: doesn't work when arr is a num.ndarray
+    assert arr[offset] == 1
```

Comment:
Actually, can we do this check as follows?
```
assert np.array_equal(arr[offset], 1)
```
This way we also check that a scalar is returned, whereas the original `assert arr[offset] == 1` will succeed if `arr[offset]` returns an array of shape `(1,)`.

Reply:
done